Islandora-CLAW/Crayfish
# Port work of @jordandukart in PR #144

**GitHub Issue**: n/a

# What does this Pull Request do?

Repeats the work @jordandukart did in #144 for all the microservices now that they all use `symfony/flex`

# What's new?
* Does this change require documentation to be updated? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (i.e. Regeneration activity, etc.)?  no
* Could this change impact execution of existing code? no

# How should this be tested?

This really should have no descernible impact, other than when `composer install`ing it you will not get a prompt to allow `symfony/flex` to execute code.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers
